Watch Extension/SessionCoordinator.swift

```diff
@@ -33,4 +34,28 @@ extension SessionCoordinator: WCSessionDelegate {
             store?.send(.session(.loggedOut))
         }
     }
+
+    func session(_ session: WCSession, didReceiveMessageData messageData: Data, replyHandler: @escaping (Data) -> Void) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c7106bd91c271d949fbb3c9cc499512408cb0c64/Watch Extension/SessionCoordinator.swift#L38"></span>Line should be 120 characters or less: currently 121 characters<br />
<code>line_length</code> <code>SessionCoordinator.swift:38</code></td>
    </tr>
  </tbody>
</table>